modules/troubleshooting-dspa-component-errors.adoc

```diff
@@ -0,0 +1,221 @@
+:_module-type: REFERENCE
+
+[id="troubleshooting-dspa-component-errors_{context}"]
+= Troubleshooting DSPA component errors
+
+[role='_abstract']
+This table displays common errors found in DataSciencePipelinesApplication (DSPA) components, along with their statuses, messages, and proposed solutions. The **Ready** condition type accumulates errors from various DSPA components, providing a status view of the DSPA deployment.
+
+[cols="10%,10%,20%,60%", options="header",]
+|===
+|Type |Status |Error message |Solution
+
+|`ObjectStorageAvailable`
+
+Ready
+|False
+
+False
+|`Could not connect to Object Store: tls: failed to verify certificate: x509: certificate signed by unknown authority`
+|This issue occurs in clusters that use self-signed certificates with {productname-short} version 2.9 or later. The data science pipelines manager cannot connect to the object storage as it does not trust the object storage SSL certificate. Therefore, the pipeline server cannot be created. 
+
+Contact your IT operations administrator to add the relevant Certificate Authority bundle. 
+
+ifdef::upstream[]
+For more information, see link:{odhdocshome}/installing-open-data-hub/#understanding-certificates_certs[Understanding certificates in {productname-short}].
+endif::[]
+
+ifdef::self-managed,cloud-service[]
+ifndef::disconnected[]
+For more information, see link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}/working-with-certificates_certs[Working with certificates].
+endif::[]
+
+ifdef::disconnected[]
+For more information, see link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}_in_a_disconnected_environment/working-with-certificates_certs[Working with certificates].
+endif::[]
+endif::[]
+
+|`ObjectStorageAvailable`
+
+Ready
+|False
+
+False
+|`Could not connect to Object Store
+Deployment for component "ds-pipeline-pipelines-definition" is missing - pre-requisite component may not yet be available.
+Deployment for component "ds-pipeline-persistenceagent-pipelines-definition" is missing - pre-requisite component may not yet be available.
+Deployment for component "ds-pipeline-scheduledworkflow-pipelines-definition" is missing - pre-requisite component may not yet be available.`
+|In clusters running RHOAI 2.8.x, the Data Science Pipelines Manager may fail to connect to the object storage, and the Pipeline Server may not be created. This can happen for several reasons.
```

Comment:
|In clusters running {productname-short} 2.8.x,

Reply:
pipeline server should be lower case too, so i've changed that and used your suggestion.

Reply:
I've also removed "This can happen for several reasons", as it's too unclear unless each of the several reasons are listed.